cumulus/ansible/tasks/providers/ec2.py

```diff
@@ -44,15 +44,18 @@ def __init__(self, profile):
             setattr(self, key, value)
 
         if not hasattr(self, 'secretAccessKey'):
-            try:
-                # Import the model from the girder plugin
-                from cumulus_plugin.models.aws import Aws as AwsModel
-                profile = AwsModel().load(self.girder_profile_id)
-
-                self.secretAccessKey = profile.get('secreteAccessKey', None)
-            # Cumulus plugin libraries are not available
-            except ImportError:
-                self.secretAccessKey = None
+            self.secretAccessKey = None
+            # Try to get the key if we have a profile id
+            if self.girder_profile_id:
```

Comment:
@cjh1 I needed to add a check here to make sure `self.girder_profile_id` is not `None` to get the tests to pass. If it was `None`, and I tried to load it with `AwsModel().load(self.girder_profile_id)`, I would get an error: `ValidationException: Attempt to load null ObjectId: None`. The secret access key is just set to `None` if `self.girder_profile_id` is `None`.